packages/oui-header-tabs/README.md

```diff
@@ -97,6 +97,7 @@
 | `active`      | boolean   | <?        | no                | `true`, `false`   | `false`   | manual active flag
 | `disabled`    | boolean   | <?        | yes               | `true`, `false`   | `false`   | disabled flag
 | `external`    | boolean   | <?        | yes               | `true`, `false`   | `false`   | external link flag
+| `on-click`    | function  | &         | no                | n/a               | n/a       | handler triggered when clicked
```

Comment:
You can simplify the description this way : `click handler` or `click event handler`

Reply:
I like `click handler` better :)